awx/sso/views.py

```diff
@@ -46,25 +45,9 @@ class CompleteView(BaseRedirectView):
     def dispatch(self, request, *args, **kwargs):
         response = super(CompleteView, self).dispatch(request, *args, **kwargs)
         if self.request.user and self.request.user.is_authenticated():
-            request_hash = AuthToken.get_request_hash(self.request)
-            try:
-                token = AuthToken.objects.filter(user=request.user,
-                                                 request_hash=request_hash,
-                                                 reason='',
-                                                 expires__gt=now())[0]
-                token.refresh()
-                logger.info(smart_text(u"User {} logged in".format(self.request.user.username)))
-            except IndexError:
-                token = AuthToken.objects.create(user=request.user,
-                                                 request_hash=request_hash)
-                logger.info(smart_text(u"User {} logged in".format(self.request.user.username)))
-            request.session['auth_token_key'] = token.key
-            token_key = urllib.quote('"%s"' % token.key)
-            response.set_cookie('token', token_key)
-            token_expires = token.expires.astimezone(utc).strftime('%Y-%m-%dT%H:%M:%S')
-            token_expires = '%s.%03dZ' % (token_expires, token.expires.microsecond / 1000)
-            token_expires = urllib.quote('"%s"' % token_expires)
-            response.set_cookie('token_expires', token_expires)
+            auth.login(self.request, self.request.user)
+            logger.info(smart_text(u"User {} logged in".format(self.request.user.username)))
+            # TODO: remove these 2 cookie-sets after UI removes them
```

Comment:
Lets not forget to do this after the UI work.

Reply:
check: removed